monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/pages/Datasources.js

```diff
@@ -20,38 +20,30 @@
 
 import React from 'react';
 import EnhancedTable from '../commons/EnhancedTable';
+import { useSelector } from 'react-redux';
+import HTTPClient from '../utils/HTTPClient';
 
-export default class Datasources extends React.Component {
-    constructor(props){
-        super(props)
-        this.state = { pageInfo: {
-                pageId: "datasources",
-                title: "Data Sources",
-                headCells: [
-                    {id: 'name', label: 'Inbound Endpoint Name'},
-                    {id: 'nodes', label: 'Nodes'},
-                    {id: 'type', label: 'Type'},
-                    {id: 'data_source_status', label: 'Status'}],
-                tableOrderBy: 'service'
-            },
-            datasourceList: [{
-                name: "Calculator EP",
-                nodes: [
-                    { nodeId: "node_01",
-                        type: "Type",
-                        data_source_status: "Inactive"
+export default function DataSources() {
+    const [pageInfo] = React.useState({
```

Comment:
Shall we refactor these `React.useState()` to `useState()` (by fixing imports) to maintain consistency. I mean all the other hooks are called like `useSelector()`, so it's better to use the same format for the `useState()` as well.

Reply:
`useEffect()` in line 42 need the same modification.

Reply:
Fixed with 795cc4d